templates/CRM/common/notifications.tpl

```diff
@@ -1,7 +1,10 @@
 <div id="crm-notification-container" role="alert" aria-live="assertive" aria-atomic=”true” style="display:none">
-  <div id="crm-notification-alert" class="#{ldelim}type{rdelim}">
-    <div class="icon ui-notify-close" title="{ts}close{/ts}"> </div>
-    <a class="ui-notify-cross ui-notify-close" href="#" title="{ts}close{/ts}">x</a>
+  <div id="crm-notification-alert" class="civi-notification-#{ldelim}type{rdelim}">
```

Comment:
this is good and you're right it will prevent clashes with other themes...
buuuut...
there may be some code out there that actually expects these classes.
Here's one such place: https://github.com/civicrm/civicrm-core/blob/master/js/Common.js#L1279

Reply:
To be fair, that's the only spot I could find, so maybe we're good changing that :)

Reply:
No wait, there's also this oddball: https://github.com/civicrm/civicrm-core/blob/master/templates/CRM/Admin/Page/Extensions.tpl#L112 which actually relies on the "type" being passed through directly to a css class.
IMO we could clean that up by adding a "loading" case to the switch above and give the icon a `fa-spin` or `fa-pulse` class.

Reply:
And by "we" I mean @agh1 ;)

Reply:
I'm working on it--I actually found a couple more places where this `crm-msg-loading` business comes up, so it'll take a little bit to figure out.